Testscripts/Windows/PERF-NETWORK-TCP-LATENCY-LAGSCOPE.ps1

```diff
@@ -118,33 +119,52 @@ collect_VM_properties
         if ($testResult -eq "PASS") {
             Write-LogInfo "Generating the performance data for database insertion"
             $properties = Get-VMProperties -PropertyFilePath "$LogDir\VM_properties.csv"
-            $resultMap = @{}
-            if ($properties) {
-                $resultMap["GuestDistro"] = $properties.GuestDistro
-                $resultMap["HostOS"] = $properties.HostOS
-                $resultMap["KernelVersion"] = $properties.KernelVersion
-            }
-            $resultMap["HostType"] = "Azure"
-            $resultMap["HostBy"] = $CurrentTestData.SetupConfig.TestLocation
-            $resultMap["GuestOSType"] = "Linux"
-            $resultMap["GuestSize"] = $clientVMData.InstanceSize
-            $resultMap["IPVersion"] = "IPv4"
-            $resultMap["ProtocolType"] = "TCP"
-            $resultMap["TestCaseName"] = $global:GlobalConfig.Global.$TestPlatform.ResultsDatabase.testTag
-            $resultMap["TestDate"] = $(Get-Date -Format yyyy-MM-dd)
-            $resultMap["LISVersion"] = "Inbuilt"
+            $testDate = $(Get-Date -Format yyyy-MM-dd)
             if ($currentTestData.SetupConfig.Networking -imatch "SRIOV") {
-                $resultMap["DataPath"] = "SRIOV"
+                $dataPath = "SRIOV"
             } else {
-                $resultMap["DataPath"] = "Synthetic"
+                $dataPath = "Synthetic"
+            }
+
+            $histogramFlag = $false
+            foreach ($line in $lagscopeReportLog) {
+                if ($line -imatch "Interval\(usec\)") {
+                    $histogramFlag = $true
+                    continue;
+                }
+                if ($histogramFlag -eq $false) {
+                    continue;
+                }
+                $interval = ($line.Trim() -replace '\s+',' ').Split(" ")[0]
```

Comment:
Can we add some comments on the format of CSV and what is this condition ?

Reply:
Added comment "# From the line 'Interval(usec)  Frequency', we begin to collect the histogram data" in line 131.